Dockerfile

```diff
@@ -1,20 +1,26 @@
 # syntax=docker/dockerfile:1.2
 
+ARG ALPINE_VERSION=latest
 ARG BUILDX_VERSION=latest
 ARG DOCKER_VERSION=latest
 
 FROM docker/buildx-bin:$BUILDX_VERSION as buildx_bin
 
+FROM alpine:$ALPINE_VERSION as buildx_strip
+
+COPY --from=buildx_bin /buildx /
+RUN apk add -U binutils && strip /buildx
```

Comment:
Not sure we need to strip the binary (doesn't remove much)

But the other updates seem useful, maybe a different pr?

Reply:
With your latest design on main, I reckon these added 4 lines only for the strip are quite a lot "bang for the buck/line".
As for the other updates, I think there is only line 19 left if the strip part is taken away, right? (And that removal of superfluous double quotes that should be single ones they were not superfluous could be sneaked into any other PR since it is such a small fix.)